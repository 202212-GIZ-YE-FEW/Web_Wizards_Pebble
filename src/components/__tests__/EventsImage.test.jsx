import renderer from "react-test-renderer";

import EventsImages from "@/components/events/EventCard/EventsImage";

it("renders correctly", () => {
    const tree = renderer.create(<EventsImages />).toJSON();
    expect(tree).toMatchSnapshot();
});
