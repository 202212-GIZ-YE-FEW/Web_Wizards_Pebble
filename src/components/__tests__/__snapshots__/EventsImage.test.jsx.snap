// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="border-2 rounded-lg overflow-hidden h-28 w-48 lg:h-40 lg:w-64 sm:h-36 sm:w-58"
>
  <img
    alt="Card image"
    className="object-fill h-28 w-48 lg:h-40 lg:w-64 sm:h-36 sm:w-58"
  />
</div>
`;
